pkg/asset/installconfig/powervs/validation.go

```diff
@@ -2,7 +2,6 @@ package powervs
 
 import (
 	"context"
-	"errors"
```

Comment:
This is the built-in `errors` lib, you don't need to remove this one.

Reply:
If I leave it in, then I get:
```
pkg/asset/installconfig/powervs/validation.go:5:2: "errors" imported and not used
```
And I would think that if I had used it in that file, then the compiler would complain if I didn't include it...